dev-libs/pybind11/pybind11-2.4.3.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Seamless operability between C++11 and Python"
+DESCRIPTION="pybind11 is a lightweight header-only library that exposes C++ \
+types in Python and vice versa, mainly to create Python bindings of existing \
+C++ code.
+Its goals and syntax are similar to the excellent Boost.Python library by \
+David Abrahams: to minimize boilerplate code in traditional extension modules \
+by inferring type information using compile-time introspection."
+HOMEPAGE="https://pypi.org/project/pybind11/"
+COPYRIGHT="2015-2019 Wenzel Jakob"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/pybind/pybind11/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="1eed57bc6863190e35637290f97a20c81cfe4d9090ac0a24f3bbf08f265eb71d"
+SOURCE_FILENAME="pybind11-v$portVersion.tar.gz"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
```

Comment:
please remove $BUILD_PREREQUIRES

Reply:
Still working on it, should also be "any" as there is no compiler involved? (at least it's working that   way, wip) :)

Reply:
Updated ...